packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -323,16 +299,11 @@ export class ObjectSetListenerWebsocket {
       // if we are already done, we don't need to do anything
       return;
     }
-
     sub.status = newStatus;
     // make sure listeners do nothing now
     sub.listener = fillOutListener<Q, any>({});
-    if (sub.expiry) {
-      clearTimeout(sub.expiry);
-      sub.expiry = undefined;
-    }
     this.#subscriptions.delete(sub.subscriptionId);
-
+    this.#sendSubscribeMessage();
```

Comment:
Why was this added here?

Reply:
To unsubscribe from an object set, you need to send another subscribe request without the given subscription request

Reply:
I see, so the old code kept getting updates (that were dropped because we reset the listener to noop) but this pre-emptively unsubscribes to stop the network traffic.